src/main/kotlin/org/jitsi/rtp/util/RtpUtils.kt

```diff
@@ -82,13 +82,21 @@ class RtpUtils {
          * @return the delta between two RTP sequence numbers (modulo 2^16).
          */
         @JvmStatic
-        fun getSequenceNumberDelta(a: Int, b: Int): Int {
+        fun getSequenceNumberDelta(a: Int, b: Int): Int =
+            getSequenceNumberDeltaAsShort(a, b).toInt()
+
+        /**
+         * Like [getSequenceNumberDelta], but returning the delta as a [Short].
+         *
+         * TODO: since [Short] can fully represent sequence number deltas,
+         *   change this to be the public API?
+         */
+        private inline fun getSequenceNumberDeltaAsShort(a: Int, b: Int): Short {
```

Comment:
```
private inline fun getSequenceNumberDeltaAsShort(a: Int, b: Int): Short = (a - b).toShort()
```
?

Reply:
I wrote out the val's for debuggability, but I agree this is probably more readable as code now that it's working.